### ISSUE TITLE
Create a page showing all publications about OBO papers (and any papers that cite them)

### ISSUE THREAD

**Issue by cmungall** [OBOFoundry/OBOFoundry.github.io#220]
It should be easy to take the publications field, e.g.

```
publications:
  - id: http://www.ncbi.nlm.nih.gov/pubmed/22293552
    title: "Uberon, an integrative multi-species anatomy ontology"
  - id: http://www.ncbi.nlm.nih.gov/pubmed/25009735
    title: "Unification of multi-species vertebrate anatomy ontologies for comparative biology in Uberon"
```

aggregate these into a single object in _config.yml and have a page that shows all publications.

Could also do a scholar/ncbi query that shows all publications citing OBO ones

We may need a separate metadata field for papers about the OBO Foundry itself - e.g. the NBT one but also associated tooling papers

**Comment by jamesaoverton** [comment]
Is there a YAML field for the single current, primary, preferred citation for an ontology?

**Comment by cmungall** [comment]
At the moment it's an undifferentiated list.

Option one is one field 'preferred_publication', card{0,1}, and keep the 
publications field a list

Another is to add a subfield for each entry in the list. A bit more 
awkward for the jsonld/rdf translation

**Comment by cmungall** [comment]
Should we have a "cite this ontology" link on every ontology page?

Alternatively, this is something that groups can format for themselves in the markdown for their ontology page

**Comment by jamesaoverton** [comment]
@cmungall Melissa asked for citation information for each ontology here: #227 

Either way, the problem is consistency. It would be hugely valuable to have someone review all the metadata files in the registry. It would also be a huge amount of work.

**Comment by zhengj2007** [comment]
I like Chris' idea. So, the ontology owners can add/update the citation of their ontology. Then the script can retrieve the information for each ontology metadata file. The review of each ontology metadata can be assigned to the ontology owners.

**Comment by mellybelly** [comment]
@zhengj2007 that was the idea. Should be 1-3 key publications for each ontology.

**Comment by jamesaoverton** [comment]
I think this is closed by PR #233, merged with commit 55ca958710f6828559a53b647b8a082f62695bb2
